app/src/main/res/values-es/strings.xml

```diff
@@ -28,4 +28,13 @@
     <string name="crime_photo_image_description">Foto de la escena del crimen</string>
     <string name="send_report">Enviar el informe del crimen a través de</string>
     <string name="error">Ocurrió un error</string>
+    <string name="create_account">CREAR UNA QUENTA</string>
```

Comment:
Leave this lower caps and use https://developer.android.com/reference/android/widget/TextView.html#attr_android:textAllCaps

Reply:
Same for the other fields below.